ONSdigital/eq-author
# 116 Webpage title

### What is the context of this PR?

[Trello card](https://trello.com/c/blvgnemC/116-webpage-title-page-name-survey-name-dahl-3)

Adds a proper meta title to the page for display in browser tabs/history, etc.

##### Bonus content

Fixed issue with main content layer not being centered consistently. Added a `MainCanvas` component to control this layout.

### How to review 

- Navigate every page and check page title against the AC in the user story.
- Check layout now matches design
